### PR TITLE
configure for installation

### PR DESCRIPTION
**Pull request by steviedale**
Added install() methods to CMakeLists.txt files

### CODE REVIEW

File: mesh_segmenter/CMakeLists.txt [hunk 1]
```diff
@@ -29,3 +29,15 @@ catkin_add_gtest(${PROJECT_NAME}-test test/utest.cpp)
...
+#############
+## Install ##
+#############
+install(
+  DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+
+install(
+  TARGETS mesh_segmenter
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
```
Comment (marip8) [inline mesh_segmenter/CMakeLists.txt]: I believe shared library targets need to be installed in both the library and runtime destinations. Typically what I've done in the past is used a single `install` command to install all libraries and executables:
```cmake
install(
  TARGETS
    target_1
    target_2
  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
)
```

**Review by marip8 — CHANGES_REQUESTED** [review]
Looks good overall. Just make the couple changes to the target install lines and I think we should be good to go.

File: mesh_segmenter/CMakeLists.txt (hunk 1 above)
Comment (steviedale) [reply mesh_segmenter/CMakeLists.txt]: Sounds good, I'll make those changes.

Comment (steviedale) [reply mesh_segmenter/CMakeLists.txt]: Changes have been made.

**Review by marip8 — APPROVED** [review]

**Pull request merged**